tensorly/decomposition/candecomp_parafac.py

```diff
@@ -80,6 +81,17 @@ def initialize_factors(tensor, rank, init='svd', svd='numpy_svd', random_state=N
             factors.append(factor)
         return factors
 
+    elif isinstance(init, (tuple, list, KruskalTensor)):
```

Comment:
Just a note: the addition of a manual initialization to the CP is also discussed in #162. I think the solution you proposed is in line with the discussions in that PR, but the syntax is a little bit different.

I am not sure about using the "init" keyword to pass the factors; this can be a little bit confusing. I suggested to use a keyword "given" and then to pass the initial factors in another argument. Not sure which is best, yours is more compact for sure.

Reply:
Just following up -- the consensus is to go with your strategy, it is probably the cleanest. We can add an intermediate function to do more fine-grained initializations later.

Reply:
We chose this option because when working on a personal implementation, I found that requiring two inputs led to a cumbersome workflow. However, both are fine, so we can change it. Should we change it here or do you prefer a separate pull request?

Reply:
I meant that your solution seems to be the best :) 
Let's keep that!